src/main/java/com/gruzini/tennistico/repositories/MatchRepository.java

```diff
@@ -15,11 +17,16 @@
 
     List<Match> findByEndingAtBeforeAndMatchStatus(LocalDateTime endingDateTime, MatchStatus matchStatus);
 
-    List<Match> findAllByMatchStatusAndPlayersNotContainsAndStartingAtIsAfterOrderByStartingAt(MatchStatus status, Player player, LocalDateTime now);
+    //metoda wyciaga mecze po statusie wg playera ktorego w meczu nie ma
+    List<Match> findAllByMatchStatusAndAndHostNotAndStartingAtAfterOrderByStartingAt(MatchStatus status, Player player, LocalDateTime now);
+    //List<Match> findAllByMatchStatusAndPlayersNotContainsAndStartingAtIsAfterOrderByStartingAt(MatchStatus status, Player player, LocalDateTime now);
 
-    List<Match> findAllByPlayersContainsAndStartingAtIsAfterAndMatchStatusInOrderByStartingAt(Player player, LocalDateTime now, List<MatchStatus> matchStatusList);
+    //metoda wyciaga mecze playera
+    List<Match> findAllByHostOrGuestAndStartingAtAfterAndMatchStatusInOrderByStartingAt(Player player, Player player1, LocalDateTime now, List<MatchStatus> matchStatusList);
+    //List<Match> findAllByPlayersContainsAndStartingAtIsAfterAndMatchStatusInOrderByStartingAt(Player player, LocalDateTime now, List<MatchStatus> matchStatusList);
```

Comment:
te komentarze wylatujo potem, co nie? :)

Reply:
tak wylecą. Na razie zostawiłem ponieważ jeszcze nie stestowałem tej metody czy prawidłowo wyciąga z bazy. Zauważ że występuje tam zarówno AND jaki OR. Nie wiem czy hibernate ogarnia pierwszeństwo w taki sposób w jaki byśmy chcieli